backend-api/src/functions/services/session/tests/sessionService.test.ts

```diff
@@ -89,11 +88,29 @@ describe("Session Service", () => {
             },
           ],
         });
-
         const result = await sessionService.getActiveSessionId("mockSub");
 
         expect(result.isError).toBe(false);
         expect(result.value).toEqual("mockSessionId");
+        const expectedCommandInput: QueryCommandInput = {
+          ExpressionAttributeValues: {
+            ":currentTimeInSeconds": { N: "1710028800" },
```

Comment:
Nit: is it possible please to add this value in a comment next to this line?

```jest.useFakeTimers().setSystemTime(new Date("2024-03-10"))```

Reply:
Done, but for all of them!